api/src/main/java/org/pmiops/workbench/genomics/GenomicExtractionService.java

```diff
@@ -313,28 +313,36 @@ private List<String> getFailureCauses(FirecloudSubmission firecloudSubmission) {
         .orElse(ImmutableList.of("unknown cause"));
   }
 
-  public GenomicExtractionJob submitGenomicExtractionJob(DbWorkspace workspace, DbDataset dataSet)
-      throws ApiException {
-    WgsCohortExtractionConfig cohortExtractionConfig =
-        workbenchConfigProvider.get().wgsCohortExtraction;
-
-    RawlsWorkspaceDetails fcUserWorkspace =
-        fireCloudService.getWorkspace(workspace).get().getWorkspace();
+  private Map<String, String> getWorkflowInputs(
```

Comment:
Here's the main change: split up the main submitGenomicExtractionJob into 2 methods because a large portion of it was assembling the inputs.

Reply:
no changes.  just rearranging for functional separation